bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2646,8 +2650,7 @@ void ReplicaImp::onMessage<ReqMissingDataMsg>(ReqMissingDataMsg *msg) {
   const ReplicaId msgSender = msg->senderId();
   SCOPED_MDC_SEQ_NUM(std::to_string(msgSeqNum));
   LOG_INFO(GL, "Received ReqMissingDataMsg. " << KVLOG(msgSender, msg->getFlags()));
-  if ((currentViewIsActive()) && (msgSeqNum > strictLowerBoundOfSeqNums) && (mainLog->insideActiveWindow(msgSeqNum)) &&
-      (mainLog->insideActiveWindow(msgSeqNum))) {
+  if ((currentViewIsActive()) && (msgSeqNum > strictLowerBoundOfSeqNums) && (mainLog->insideActiveWindow(msgSeqNum))) {
```

Comment:
After a wedge, no client commands are every processed, right?

Is it ever possible that a view becomes inactive somehow after a wedge, and therefore this condition doesn't hold. 

I think a way to test it would be to partition a replica at the right time, send some client requests, wait for view change timeout, then see if you can rejoin the replica and reach N/N. Maybe this is overkill though.

Reply:
I don't think it is possible, but anyway, even if it is, it doesn't matter. The system is wedged only w.r.t running consensus on new client request.
Vire change still can happen on the system

Reply:
Ah ok. Cool. Thank you!